apps/common/lib/lexical/ast.ex

```diff
@@ -15,10 +65,13 @@ defmodule Lexical.Ast do
 
   @type t :: any()
```

Comment:
Can we use another `type` instead of `any`?

Reply:
Yeah, maybe we should just use `Macro.t()` directly instead and remove `Ast.t()`. That's what Sourceror does, for instance, which is a good example to follow. WDYT?

Reply:
I think `Macro.t()` is good.